Game/Source/GDKTestGymsFunctionalTests/Tests/PredictedGameplayCuesTest/GA_ApplyCueEffect.cpp

```diff
@@ -29,7 +29,7 @@ void UGA_ApplyCueEffect::ActivateAbility(const FGameplayAbilitySpecHandle Handle
 	{
 		if (!CommitAbility(Handle, ActorInfo, ActivationInfo))
 		{
-			EndAbility(CurrentSpecHandle, CurrentActorInfo, CurrentActivationInfo, false, true);
+			EndAbility(Handle, ActorInfo, ActivationInfo, false, true);
```

Comment:
Ultra nit and I have no idea what I am looking at:
Is this function now only dependent on input args? Is it static?

Reply:
Yes on only being dependent on input args, but no on the static bit. This is an overridden function from UGameplayAbility, so we can't change anything about the signature.